wordpress-mobile/WordPress-Android
# Issue/13333 add p2 site icon

Fixes #13333

This PR replaces a default globe site icon placeholder for P2 sites.

I replaced it in all the areas where it makes sense, including Reader.

@mattmiklic I'm not sure what was your intention regarding the colors of the P2 icon, so I just matched them to the current ones.

[![Image from Gyazo](https://i.gyazo.com/2c846db37872b06b93787fd020258213.png)](https://gyazo.com/2c846db37872b06b93787fd020258213)

[![Image from Gyazo](https://i.gyazo.com/a32c4a34dd25ee3ccb2f716a7744bdd4.png)](https://gyazo.com/a32c4a34dd25ee3ccb2f716a7744bdd4)

[![Image from Gyazo](https://i.gyazo.com/0c26b09e14c980625a017d4eea467c01.png)](https://gyazo.com/0c26b09e14c980625a017d4eea467c01)

To test:
- Make sure you have a P2 site without an icon and you also follow it in Reader.
- Check the icon of the P2 site in My Site, Site Picker, and Reader Blog Details and make sure it looks good (you might need to reinstall the app to refresh posts in reader since we added a P2 filed to the DB table)

PR submission checklist:
